v2.0.1/admin-schema.json

```diff
@@ -0,0 +1,692 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v2.0.1/admin-schema.json",
+    "title": "Schema for Modeling Hub administrative settings",
+    "description": "This JSON file provides a schema for modeling hub administrative information.",
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "description": "URL to a version of the Modeling Hub schema admin-schema.json file (see https://github.com/Infectious-Disease-Modeling-Hubs/schemas). Used to declare the schema version an 'admin.json' file is written for and for config file validation. The URL provided should be the URL to the raw content of the schema file on GitHub.",
+            "examples": ["https://raw.githubusercontent.com/Infectious-Disease-Modeling-Hubs/schemas/main/v0.0.1/admin-schema.json"],
+            "type": "string",
+            "format": "uri"
+        },
+        "name": {
+            "description": "The name of the hub.",
+            "type": "string",
+            "examples": ["US COVID-19 Forecast Hub​"]
+        },
+        "maintainer": {
+            "description": "The entity that maintains and runs the hub.",
+            "type": "string",
+            "examples": ["The Consortium of Infectious Disease Modeling Hubs"]
+        },
+        "contact": {
+            "description": "The name and email of a human being who serves as a point of contact for the hub.",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "email": {
+                    "type": "string",
+                    "format": "email"
+                }
+            }
+        },
+        "repository_host": {
+            "description": "The name of the web host for the hub repository",
+            "type": "string",
+            "enum": [
+                "GitHub"
+            ]
+        },
+        "repository_url": {
```

Comment:
Since we know that all hubs are using GitHub, I don't think it's worth making a breaking change here, but if we ever hit v3.0, it might be helpful to split up the `repository_url` into its atomic components of org name and repo name.

Reason: cloud-enabled hubs need the org name and repo name as separate strings for setting up AWS permissions. As we ramp up, it's easy enough to parse because we know how GitHub URLs are formatted.

Reply:
This seems like a reasonable suggestion. maybe worth filing as an issue for future reference?

Reply:
https://github.com/Infectious-Disease-Modeling-Hubs/schemas/issues/67